node/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    extends: 'eslint:recommended',
```

Comment:
Why this file isn't affected by eslint rules (4 spaces instead of 2)?

Reply:
resolved